src/sql/workbench/parts/dashboard/widgets/properties/propertiesWidget.component.ts

```diff
@@ -18,11 +20,32 @@ import * as types from 'vs/base/common/types';
 import * as nls from 'vs/nls';
 import { Registry } from 'vs/platform/registry/common/platform';
 import { ILogService } from 'vs/platform/log/common/log';
+import { IEndpoint } from 'sql/workbench/parts/notebook/notebookUtils';
 
 export interface PropertiesConfig {
 	properties: Array<Property>;
 }
 
+export interface EndpointsConfig {
```

Comment:
>export [](start = 0, length = 6)

Does this need to be exported?

Reply:
Same goes for all of these - they don't appear to be used outside this file

---
In reply to: [282330086](https://github.com/microsoft/azuredatastudio/pull/5386#discussion_r282330086) [](ancestors = 282330086)

Reply:
Well, it's not my design. This was already there and am re-using this widget for the endpoints. Probably a good question. But if they are re-usable, I guess it's fair to have them as exported.

Reply:
Not a huge deal - but I prefer the principal of least-visibility. Don't expose something unless you need to. Helps keep the testable surface area down and increases safety when the scope is contained within the same file.